pkgs/tools/archivers/7zz/default.nix

```diff
@@ -117,6 +119,8 @@ stdenv.mkDerivation (finalAttrs: {
       package = finalAttrs.finalPackage;
       command = "7zz --help";
     };
+    # used by update script
+    unfree = _7zz.override { enableUnfree = true; };
```

Comment:
also, this is probably fine for the update script, but we should probably note that `_7zz` should never be used anywhere else in the file

Reply:
Resolved by including an older commit of mine that exposes the value in top-level.